metaflow/cards.py

```diff
@@ -8,6 +8,7 @@
     Markdown,
     VegaChart,
     ProgressBar,
+    PythonCode,
```

Comment:
Didn't check but is this just for python code or can it handle any ```<language> type of syntax?

Reply:
No just python code. There is no other code because the `prism` library vendored in our cards package currently only has support for `python`, `log` and `c-like` languages. We would have to vendor other languages for full support (but that was too much work for now and we didn't face any usecases for such)